tools/matrix-bot/logger.ts

```diff
@@ -22,24 +17,49 @@ export class MessagesLogger {
     if (!eventId) {
       return;
     }
-    if (!msg.includes("graypaper.fluffylabs.dev")) {
+    const gpLinks = findLinks(msg);
+    if (!gpLinks.length) {
       return;
     }
 
+    // if there are several links in the message, find the link with the latest version
+    let versionName: string | undefined;
+
+    if (gpLinks.length === 1) {
+      versionName = parseLink(gpLinks[0], this.meta)?.versionName;
+    } else {
+      versionName = gpLinks
+        .map((link) => parseLink(link, this.meta))
+        .filter(Boolean)
+        .map((link) => link?.versionName)
+        .sort()
+        .pop();
+    }
+
+    if (!versionName) {
+      return;
+    }
+
+    const majorVersion = versionName.replace(/^(\d+\.\d+\.)\d+$/, "$1x");
```

Comment:
```suggestion
    const majorVersion = `${versionName.split('.')[0]}x`;
```
I'm not a huge fan of regex, because my brain-parser for them is pretty bad, hence the suggestion.

Reply:
that would produce something like "0x", unless im missing something? doing it properly without a regex would take more than that

Reply:
Ah, right! Then: `${versionNameParts.slice(0, 2).join('.')}-x` should work?